### DISCUSSION TITLE
cCleaner pinned to open shell menu

### DISCUSSION THREAD

**Discussion by Philipdp13** [Open-Shell/Open-Shell-Menu#1842] [Q&A] [answered]
I have Wil 11 pro on both a desktop and laptop with cCleaner pinned to the start of both.  On one menu it just shows as a shortcut, on the other it shows with an arrow and 'task' .[options.  Any ideas why this is, and how I can just show the basic shortcut on both please?
![Untitled 1](https://github.com/Open-Shell/Open-Shell-Menu/assets/117991730/87a1e736-743b-4afc-899c-0fc9135656ef)

**Comment by ge0rdi** [comment, marked as answer]
I think you have `Enable jumplists` option enabled on one of your machines.
Try to disable it.

**Comment by Philipdp13** [comment]
Thanks – I’ll give it a try tomorrow when I’m logged into the laptop again

**Comment by Philipdp13** [comment]
Yes – that solved it for me – thank you!
